src/core/os/amdgpu/amdgpuScreen.cpp

```diff
@@ -157,11 +151,6 @@ Result Screen::GetFormats(
     const uint32 baseFormatCount = sizeof(PresentableSwizzledFormat) / sizeof(PresentableSwizzledFormat[0]);
```

Comment:
Please remove baseFormatCount and set formatCount with sizeof(PresentableSwizzledFormat) / sizeof(PresentableSwizzledFormat[0]); directly.

Reply:
I can do that. However, my next pull request, immediately after acceptance of this one, will be for adding fp16 displayable swapchain image support, and it will refactor things again. See the already successfully tested branch here (tested on Raven Ridge/DCN1 and Polaris11 DCE11.2) with Linux 5.8-rc7:

https://github.com/kleinerm/pal/tree/fp16colorssdrhdr

Should i still make the change on this one, before sending the new fp16 out?

Reply:
Never mind, i pushed the change.